### ISSUE TITLE
How to judge if my model has been overfitted when training?

### ISSUE THREAD

**Issue by B1D1ng** [AlexeyAB/darknet#6644]
How can I tell if the model has been fitted? In my obj.data file, train=./train.txt valid=./test.txt. Now I want to know when to overfit the model during the training so as to stop the training in time. However, using the -map instruction can only draw the map curve of the test set, and no map curve of the training set can be compared to determine whether it is overfitting. At present, The method I came up with is to manually calculate the mAP of the training set for the same number of iterations according to the mAP curve in the chart.png image generated, and then compare them. However, this is undoubtedly a time-consuming work. How do you judge the overfitting when training the model? Are there code instructions to determine if the model is overfitted to stop early?
Looking forward to your reply, thank you!
The curve of mAP and loss is as follows:
![chart](https://user-images.githubusercontent.com/31911710/92838107-f8db5800-f410-11ea-9225-7d5970ba1541.png)

**Comment by sreetamadas** [comment]
Hi,
Did you find a solution to your question? Can you please post it?
Thanks

**Comment by Chitti21** [comment]
@B1D1ng 

Hi, 

Did you get the solution ?
